helm_deploy/crime-portal-gateway/templates/_envs.tpl

```diff
@@ -22,6 +22,12 @@ env:
         name: crime-portal-gateway-secrets
         key: APPINSIGHTS_INSTRUMENTATIONKEY
```

Comment:
Should this `APPINSIGHTS_INSTRUMENTATIONKEY` block be removed? It looks unused now

For future reference, we can do this without the need to create a new secret:

[see here](https://github.com/search?q=repo%3Aministryofjustice%2Fhmpps-person-record%20APPINSIGHTS_INSTRUMENTATIONKEY&type=code
)